README.md

```diff
@@ -7,19 +7,77 @@ ConfigCat is a feature flag and configuration management service that lets you s
 
 ConfigCat is a <a href="https://configcat.com" target="_blank">hosted feature flag service</a>. Manage feature toggles across frontend, backend, mobile, desktop apps. <a href="https://configcat.com" target="_blank">Alternative to LaunchDarkly</a>. Management app + feature flag SDKs.
 
-## Installation
+[![Build Status](https://travis-ci.com/configcat/elixir-sdk.svg?branch=master)](https://travis-ci.com/configcat/elixir-sdk)
+[![Coverage Status](https://codecov.io/github/configcat/elixir-sdk/badge.svg?branch=master)](https://codecov.io/github/configcat/elixir-sdk?branch=master)
+[![Hex.pm](https://img.shields.io/hexpm/v/configcat.svg?style=circle)](https://hex.pm/packages/configcat)
+[![Hex.pm](https://img.shields.io/hexpm/dt/configcat.svg?style=circle)](https://hex.pm/packages/configcat)
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed
-by adding `config_cat` to your list of dependencies in `mix.exs`:
+# Getting Started
+### 1. Add `configcat` to your list of dependencies in `mix.exs`:
```

Comment:
As I mentioned on #50, naming the library `configcat` means that we need to rename the main module as `Configcat` to follow Elixir naming conventions. I'd prefer not to do that if at all possible, because then we can use the proper name of the service in our client code (`ConfigCat`).